README.md

```diff
@@ -165,14 +165,27 @@ library
   default-language:    Haskell2010
 ```
 
-If you want to be able to import `Extra.*` modules when using `mixins` approach,
-you need to list those modules under `mixins` field as well, like this:
+This way, you can't access any other module of relude. If you want to use e.g.
+`Relude.Extra.Enum`, you need to list it (and potentially other modules) under
+the `mixins` field as well, like this:
 
 ```cabal
   mixins:              base hiding (Prelude)
                      , relude (Relude as Prelude, Relude.Extra.Enum)
 ```
 
+If you want to be able to import every module of relude, you need to add `relude`
+a second time, like this:
+
+```cabal
+  mixins:              base hiding (Prelude)
+                     , relude (Relude as Prelude)
+                     , relude
+```
+
+For more details on this `mixin` feature, see the
+[Cabal user guide](https://www.haskell.org/cabal/users-guide/developing-packages.html#pkg-field-mixins).
```

Comment:
I don't think that we need to mention Cabal user guide. We don't mention any other features (like `build-depends` or `default-extensions`) so I don't see how `mixins` are special.

Reply:
I am happy to remove it, but @vrom911 mentioned that I should avoid the word `mixin` as people might not be familiar with it, so I thought it might be good to add a link to the documentation of this feature, as it is different to e.g. `build-depends` in that it is a lot less known and is non-trivial to use (see the reason for this PR).

Reply:
It is not true. I didn't tell that. You can revisit my comment.
I said that the usage "mixin" as a verb is not clear. To be honest it is not clear to me as well, though I am familiar with this cabal feature. And this is actually the first time I see somebody using "mixin" as a verb to indicate some action.

Seems like you do not agree with something that I said and trying to play some games. If I said something that upset you, I am sorry. I didn't want to offend. All I wanted to do is to help to create better docs and improve it wherever I saw the possibility.

Reply:
I did not disagree with you that, as you said it

> using `mixin` as the verb looks fun, however could be a little too complicated for understanding for people not that fluent with the cabal terminology,

and therefore I included a link to the Cabal user guide in order to provide help for people who are not fluent with the cabal terminology, and in addition I replaced the word "mixin" (see [here]), as you suggested.

I am not offended, and I am sorry that you got the impression that I want to "play games", I also just want to add a tiny piece of documentation that made my life easier and that I wanted to share.

Reply:
Thanks for sharing your understanding. Unfortunately, this is not what I mean. There is nothing wrong with the `mixin` word. We use it when we talk about `mixin`s, it is fine. You can see that we have been using `mixin` in this document before this PR. So it is completely normal to call things by their names. `mixin` is the word that represents the feature, it is not the verb that represents the action, that is what bothers me. That what I meant. You can use (and you actually better to use) the word `mixin` to speak about `mixin`. But the verb (to mixin smth) doesn't look right to me. Sorry for the inconvenience.

Reply:
Yeah, I agree with you completely, thats why I changed it, as I wrote above.